pkg/api/apis/operators/clusterserviceversion_types.go

```diff
@@ -163,6 +246,7 @@ type ClusterServiceVersionSpec struct {
 	Maturity                  string
 	CustomResourceDefinitions CustomResourceDefinitions
 	APIServiceDefinitions     APIServiceDefinitions
+	WebhookDefinitions        []WebhookDescription
```

Comment:
typo: the slice should be `[]WebhookDefinitions` ?

Reply:
I wanted to keep it similar to `APIServiceDefinitions`, I'm not opposed to changing the variable (API) name though.